ui/src/core_components/input/CoreFileInput.vue

```diff
@@ -105,30 +133,37 @@ const encodeFile = async (file: File) => {
 
 const fileChange = async (ev: InputEvent) => {
 	const el = ev.target as HTMLInputElement;
-	if (!el.files || el.files.length == 0) return;
+	handlefiles(el.files);
+};
+
+const handlefiles = async (Files: FileList) => {
```

Comment:
All function arguments appear to be written in lowercase. Generaly, these are the types that start with a capital letter. Is there a constraint to write this one with a capital letter? Would it improve the readability of the code to use minuscule ?